modules/git/command.go

```diff
@@ -198,9 +199,6 @@ func (c *Command) RunInDirTimeoutEnv(env []string, timeout time.Duration, dir st
 		return nil, ConcatenateError(err, stderr.String())
 	}
 
-	if stdout.Len() > 0 {
-		log("stdout:\n%s", stdout.Bytes()[:1024])
-	}
```

Comment:
Could replace this with a trace level:

```suggestion
if stdout.Len() > 0 && log.IsTrace() {
  log.Trace("Stdout:\n %s", stdout.Bytes()[:1024])
}
```

The output of the git command has come in handy occasionally and Trace is probably the level it should come out at.

Reply:
(we need the log.IsTrace() test becase we should only bother with sending the message if it could possibly be logged.)